File: Latihan B/Python/Processor.py
class Processor():

	#atribut private
	iName = "" #nama prosesor
	iPrice = 0 #harga prosesor


	#konstruktor
	def __init__(self, iName = "", iPrice = 0):
		self.iName = iName
		self.iPrice = iPrice

	#getter dan setter atribut iName
	def setName(self, iName):
		self.iName = iName

	def getName(self):
		return self.iName

	#getter dan setter atribut iPrice
	def setPrice(self, iPrice):
		self.iPrice = iPrice

	def getPrice(self):
		return self.iPrice

File: Latihan A/Processor.cpp
class Processor{

	private:
		//atribut private
		string name; //nama prosesor
		int price; //harga prosesor

	public:

		//konstruktor
		Processor(){

		}

		Processor(string n, int p){
			this->name = n;
			this->price = p;
		}

		//getter dan setter atribut name
		void setName(string n){
			this->name = n;
		}

		string getName(){
			return this->name;
		}

		//getter dan setter atribut price
		void setPrice(int p){
			this->price = p;
		}

		int getPrice(){
			return this->price;
		}

		//destruktor
		~Processor(){

		}

};